docker-entrypoint.sh

```diff
@@ -50,6 +50,29 @@ if expr "$1" : "apache" 1>/dev/null; then
 fi
 
 if expr "$1" : "apache" 1>/dev/null || [ "$1" = "php-fpm" ] || [ "${NEXTCLOUD_UPDATE:-0}" -eq 1 ]; then
+    uid="$(id -u)"
+    gid="$(id -g)"
+    if [ "$uid" = '0' ]; then
+        case "$1" in
+            apache2*)
+                user="${APACHE_RUN_USER:-www-data}"
+                group="${APACHE_RUN_GROUP:-www-data}"
+
+                # strip off any '#' symbol ('#1000' is valid syntax for Apache)
```

Comment:
@J0WI Just skimming through this code and I'm wondering where this is coming from, because the base image apparently doesn't support this either (it might have in the past though)? See https://github.com/docker-library/php/blob/95a88177c3038064d1259b4fd2d4a0275201a6d7/Dockerfile-linux.template#L113. Or am I missing something?